apps/fabric-website/src/components/IconGrid/IconGrid.tsx

```diff
@@ -56,36 +58,84 @@ export class IconGrid extends React.Component<IIconGridProps, IIconGridState> {
     );
   }
 
+  public async componentDidMount() {
+    // Load the icons async after mount to avoid them being bundled and parsed with the page,
+    // due to their large bundle size.
```

Comment:
The main idea of this change was that loading all the icon files is expensive, so we shouldn't do that on initial page load. The easiest way to do the loading asynchronously was to move it to IconGrid. (It's not ideal to so tightly couple a component to specific sets of data, but I think it's okay here since IconGrid is a pretty special-purpose component anyway.)